### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/attrs.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#795] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/attrs.html

你可以使用特殊标记为 Markdown 元素添加属性。

https://theme-hope.vuejs.press/zh/guide/markdown/attrs.html

**Comment by brycenhe** [comment]
如果文档中需要使用{}，但是又不是添加属性，怎么办呢？
例如：
”期望接收数据格式为{num1, num2, ..., numN}“

**Reply by brycenhe** [reply]
找到了：： 

使用 \ 进行转义即可。

\{num1, num2, ..., numN}
